routing.rst

```diff
@@ -238,6 +238,72 @@ URL                       Route          Parameters
 ``/blog/yay-routing``     ``blog_show``  ``$slug`` = ``yay-routing``
 ========================  =============  ===============================
 
+If you prefer, requirements can be inlined in each placeholder using the syntax
+``{placeholder_name<requirements>}``. This feature makes configuration more
+concise, but it can decrease route readability when requirements are complex:
+
+.. configuration-block::
+
+    .. code-block:: php-annotations
+
+        // src/Controller/BlogController.php
+        namespace App\Controller;
+
+        use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+        use Symfony\Component\Routing\Annotation\Route;
+
+        class BlogController extends Controller
+        {
+            /**
+             * @Route("/blog/{page<\d+>}", name="blog_list")
+             */
+            public function list($page)
+            {
+                // ...
+            }
+        }
+
+    .. code-block:: yaml
+
+        # config/routes.yaml
+        blog_list:
+            path:      /blog/{page<\d+>}
+            controller: App\Controller\BlogController::list
+
+    .. code-block:: xml
+
+        <!-- config/routes.xml -->
+        <?xml version="1.0" encoding="UTF-8" ?>
+        <routes xmlns="http://symfony.com/schema/routing"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="http://symfony.com/schema/routing
+                http://symfony.com/schema/routing/routing-1.0.xsd">
+
+            <route id="blog_list" path="/blog/{page <\d+>}"
```

Comment:
Is it normal to have here a space before `<` ?

Reply:
Good catch! That was a bug. Fixed now. Thanks!